.travis.yml

```diff
@@ -1,21 +1,34 @@
 language: php
+
 php:
-  - 7.0
-  - 7.1
-  - 7.2
+  - '7.0'
+  - '7.1'
+  - '7.2'
+  - '7.3'
+  - '7.4snapshot'
```

Comment:
7.4 is out and 8.x needs top be added

Reply:
Probably Travis CI should be replaced with GH actions.